README.md

```diff
@@ -45,11 +55,11 @@ Use the [installation instructions in the Hugo documentation](http://gohugo.io/o
 
 1. Make sure your local environment has the following software installed:
 
-    * [Git](http://git-scm.com/)
-    * [Mercurial](http://mercurial.selenic.com/)
+    * [Git](https://git-scm.com/)
+    * [Mercurial](https://www.mercurial-scm.org/)
     * [Go][] 1.4+
 
-2. Fork the [Hugo project on GitHub](https://github.com/spf13/hugo).
+2. [Fork the Hugo project on GitHub](https://github.com/spf13/hugo#fork-destination-box).
```

Comment:
@bep I expected this link to open the fork box when clicked on, but it doesn't seem to do that. The previous version of this line is therefore better and less confusing.